src/crawler.ts

```diff
@@ -878,12 +885,12 @@ self.__bx_behaviors.selectMainBehavior();
     const { page, cdp, data, workerid } = opts;
     const { url } = data;
 
+    if (!data.isHTMLPage) {
```

Comment:
I think returning here means we won't wait `pageExtraDelay` after non-HTML pages